hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -381,7 +381,6 @@ object DataSourceWriteOptions {
   val DEFAULT_ASYNC_COMPACT_ENABLE_OPT_VAL = "true"
 
   // Avro Kafka Source configs
-  val KAFKA_AVRO_VALUE_DESERIALIZER = "hoodie.deltastreamer.source.kafka.value.deserializer.class"
-  // Schema to be used in custom kakfa deserializer
-  val KAFKA_AVRO_VALUE_DESERIALIZER_SCHEMA = "hoodie.deltastreamer.source.kafka.value.deserializer.schema"
+  val KAFKA_AVRO_VALUE_DESERIALIZER_OPT_KEY = "hoodie.deltastreamer.source.kafka.value.deserializer.class"
+  val DEFAULT_KAFKA_AVRO_VALUE_DESERIALIZER_OPT_VAL = "org.apache.hudi.utilities.deser.KafkaAvroSchemaDeserializer"
```

Comment:
I would prefer to make the new deserializer default in next release. by that time, I assume atleast you would have tested it and would be running it in prod and so we know it works smoothly end to end.

Reply:
Alrigth, seems perfectly fine to me. Fixed